### ISSUE TITLE
Website and Extension Policy Notice

### ISSUE THREAD

**Issue by devxan** [ScratchAddons/ScratchAddons#714]
Hello, @ScratchAddons team! I want people to know about [**THIS**](https://scratch.mit.edu/discuss/topic/284272/) policy mentioned in the scratch forum so that they don't get themselves in trouble. (https://scratch.mit.edu/discuss/topic/284272/) 
There should be a notice on the ScratchAddons options page, the [website](https://scratchaddons.com/), and the extension download page telling users about the Extension/userscript policy mentioned above/in the forums. 

This has also been posted on the other repository for @ScratchAddons, linked **[here](https://github.com/ScratchAddons/website/issues/64)**.

**Comment by j5155** [comment]
You could also just make a warning when a user is making a post in Bugs and Glitches about making sure not to mention SA and also test the bug in incognito or something where SA isn't enabled before reporting it

**Comment by TheColaber** [comment]
Next time just make one issue... Yes, we do have already a warning when creating a topic on the Bugs and Glitches forum to make sure the bug isn't from SA. In regards to your other issue, it hasn't been much of an issue in my experience, however, if you any people advertising SA, let us know and we can consider it.

**Comment by Ani-Xan** [comment]
> You could also just make a warning when a user is making a post in Bugs and Glitches about making sure not to mention SA and also test the bug in incognito or something where SA isn't enabled before reporting it
@j5155 

I mean, I agree with your idea, but I'm just talking about a little text box linking to the forums post. Adding the code and things like that would be unneeded work. 

> Next time just make one issue... Yes, we do have already a warning when creating a topic on the Bugs and Glitches forum to make sure the bug isn't from SA. In regards to your other issue, it hasn't been much of an issue in my experience, however, if you any people advertising SA, let us know and we can consider it.
@TheColaber 

I share this extension with a lot of people (just the website URL to stay within the policy guidelines) and I don't want them getting in trouble, so.... 
Sorry that I made two issues, I will delete/close the other one now. 😄

**Comment by TheColaber** [comment]
@WorldLanguages Is it allowed to advertise our website?

**Comment by wgyt735yt** [comment]
> 
> 
> Is it allowed to advertise our website?
No, thats not ok

**Comment by Ani-Xan** [comment]
> > Is it allowed to advertise our website?
> 
> No, that's not ok

Oh no! Why do you think that, @wgyt735yt?

**Comment by wgyt735yt** [comment]
> 
> 
> > > Is it allowed to advertise our website?
> > 
> > 
> > No, that's not ok
> 
> Oh no! Why do you think that, @wgyt735yt?

It would be advertising of the website, which advertises the addon itself.

**Comment by j5155** [comment]
> I share this extension with a lot of people (just the website URL to stay within the policy guidelines) and I don't want them getting in trouble, so....

You cannot, I repeat CANNOT, mention any form of extension or user script on Scratch itself. Outside of scratch, no one cares, but please do not talk about this extension or any others on scratch, not even the website URL or just the name. This github or the extension discord is the place for discussion about this extension.

**Comment by Hans5958** [comment]
We already did #552 to warn people before sending their bug reports to B&G. We also have mentioned this on the FAQ that will be released in the future.

**Comment by Ani-Xan** [comment]
> Also, opening multiple issues is not a problem to me as it covers multiple repos of the project.

@Hans5958: Ok, makes sense. (I agree.)
@wgyt735yt @j5155 Ok, just read [the whole policy forum post](https://scratch.mit.edu/discuss/topic/284272/) and you guys were right, my mistake. 😭
But I still think we should add a notice in the places above (As well as the FAQ that @Hans5958 mentioned) that tells users to not advertise the addon extension or to link to the Scratch Addons website on scratch. 

Although I still think you can link to the [ScratchAddons website](https://scratchaddons.com/contributors#contributors:~:text=Ani%2DXan) on your personal/scratch website, because you're _technically_ not mentioning it **(at all)** on Scratch. 

In conclusion, I think that you should advertise the [website](https://scratchaddons.com/contributors#contributors:~:text=Ani%2DXan) on your own sites. This is okay according to the Scratch policy, and it's highly suggested. (By me) 🎉

**Comment by j5155** [comment]
@Ani-Xan I actually just said a thing in the SA discord about adding a notice in the places above, except I said something more like a annoyance to anyone who tries. If you wanna see details feel free to join the discord

**Comment by Ani-Xan** [comment]
> @Ani-Xan I actually just said a thing in the SA discord about adding a notice in the places above, except I said something more like a**n** annoyance to anyone who tries. If you wanna see details feel free to join the discord
Here's the link for anyone who wants to see it. (Have to be invited to SA development Discord)
https://discord.com/channels/751206349614088204/751207339545198682/775873629841260564

**Comment by wgyt** [comment]
> Although I still think you can link to the [ScratchAddons website](https://scratchaddons.com/contributors#contributors:~:text=Ani%2DXan) on your personal/scratch website, because you're _technically_ not mentioning it **(at all)** on Scratch.

That is true, but if the Scratch Team found out they would probably want you to not advertise your site :|
I would still think that this notice would be a good idea, it could work by checking to see if a users post contains keywords like "addons", "scratch addons", "scratchaddons.com" or stuff like that.

**Comment by RedGuy12** [comment]
Or "extention" or maybe some of the popular addons like "dark mode"


Ofc, it wouldn't stop then from posting anything, just remind them not to.
________________________________

**Comment by CuriousG2** [comment]
People are posting a lot of things about scratchaddons on World_Languages project: TurboWarp: Scratch with dark mode, and more.

**Comment by mxmou** [comment]
Fixed by #2212? I think that's a better way to do this than a notice on the settings page.

**Comment by wgyt** [comment]
> Fixed by #2212? I think that's a better way to do this than a notice on the settings page.

I guess it is solved
